protocol/ethereum/provider/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Module,
+  manifest,
+  Args_request,
+} from "./wrap";
+import { ethers, Wallet } from "ethers";
+import { PluginFactory } from "@polywrap/core-js";
+
+export interface ProviderConfig {
+  url: string
+  wallet?: Wallet
```

Comment:
Does this mean users need to have ethers as a dependency so they can provider a `Wallet` to the config?